cmd/zed/zed_conf.c

```diff
@@ -290,6 +292,9 @@ zed_conf_parse_opts(struct zed_conf *zcp, int argc, char **argv)
 		case 'M':
 			zcp->do_memlock = 1;
 			break;
+		case 't':
+			zcp->do_testmode = 1;
+			break;
```

Comment:
This should go after the `f` case to keep the ordering the same as that of `opts`.